protocol/x/clob/keeper/deleveraging.go

```diff
@@ -305,22 +329,50 @@ func (k Keeper) OffsetSubaccountPerpetualPosition(
 	return fills, deltaQuantumsRemaining
 }
 
+// getDeleveragingQuoteQuantums returns the quote quantums delta to apply to a deleveraging operation.
+// This returns the bankruptcy price for standard deleveraging operations, and the oracle price for
+// final settlement deleveraging operations. The type of deleveraging event is determined by the
+// clob pair status of the clob pair associated with the provided perpetual.
+func (k Keeper) getDeleveragingQuoteQuantumsDelta(
+	ctx sdk.Context,
+	perpetualId uint32,
+	subaccountId satypes.SubaccountId,
+	deltaQuantums *big.Int,
+) (deltaQuoteQuantums *big.Int, err error) {
+	clobPair := k.mustGetClobPairForPerpetualId(ctx, perpetualId)
+	isFinalSettlement := clobPair.Status == types.ClobPair_STATUS_FINAL_SETTLEMENT
+
+	if isFinalSettlement {
+		deltaQuoteQuantums, err = k.perpetualsKeeper.GetNetNotional(ctx, perpetualId, deltaQuantums)
+	} else {
+		deltaQuoteQuantums, err = k.GetBankruptcyPriceInQuoteQuantums(
```

Comment:
nit: we can just `return k.perpetualsKeeper.GetNetNotional` or `return k.GetBankruptcyPriceInQuoteQuantums`. up to you

Reply:
true, will update